sdk/core/Azure.Core/samples/Serialization.md

```diff
@@ -134,12 +165,10 @@ options.GenericTypeSerializerCreator = type => type.Equals(typeof(ModelT)) ? new
 Envelope<ModelT> model = ModelSerializer.Deserialize<Envelope<ModelT>>(new BinaryData(Encoding.UTF8.GetBytes(serviceResponse)), options: options);
 ```
 
-## Using ModelSerializer with generic IModelSerializable
+## Interfacing with the low level interfaces
 
-In this example we demonstrate how we could use the exact same method / interface to serialize either json or xml.
+In this example we demonstrate how we could use the exact same method / interface to serialize either Json or Xml.
 Above we demonstrate using `SerializeJson` and `SerializeXml` methods but this requires the user to know which serializer to use.
-We could delegate needing to know this to each model itself since they for sure need to know how to serialize themselves.  It does
-introduce a question around what happens if a model could be serialized as both json and xml.
 
 ### Serialization
```

Comment:
I find the fact that we're creating an instance of something called "interface" incredibly confusing.  The term has a very specific meaning to .NET developers - and we're using it in a very different way.   I don't know if the ship has sailed on naming at this point, but I'd suggest we reconsider this one.

//cc: @m-nash, @KrzysztofCwalina